Gemfile

```diff
@@ -59,6 +59,7 @@ gem "compass-rails",  "2.0.5"
 gem "sass-rails",     "5.0.4"
 gem "autoprefixer-rails", "6.3.6.2"
 gem "bootstrap-switch-rails", "3.3.3"
+gem "font-awesome-rails", "4.4.0.0"
```

Comment:
If we officially decide to use font-awesome and drop entypo, can you open an issue about that migration? Is that what we want to do? Or to have both is also okay?